docker-compose.yml

```diff
@@ -16,9 +24,68 @@ services:
     depends_on:
       - db
     restart: unless-stopped
+    networks:
+      - monitoring
   db:
     image: "mongo:${MONGO_DB_VERSION}"
     container_name: periodum_db
     volumes:
         - ./db/mongodb:/data/db
     restart: unless-stopped
+    networks:
+      - monitoring
+  grafana:
+    image: grafana/grafana:8.2.0
+    container_name: grafana
+    volumes:
+      - grafana_data:/var/lib/grafana
+      - ./grafana/provisioning:/etc/grafana/provisioning
+    environment:
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_SERVER_HTTP_PORT=8080
+    ports:
+      - "8080:8080"
+    expose:
+      - 8080
+    networks:
+      - monitoring
+  node-exporter:
+    image: prom/node-exporter:latest
+    container_name: node-exporter
+    restart: unless-stopped
+    volumes:
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /:/rootfs:ro
+    command:
+      - '--path.procfs=/host/proc'
+      - '--path.rootfs=/rootfs'
+      - '--path.sysfs=/host/sys'
+      - '--collector.filesystem.mount-points-exclude=^/(sys|proc|dev|host|etc)($$|/)'
+    ports:
+      - "9100:9100"
```

Comment:
Let's expose those ports only `locally`. "127.0.0.1:9100:9100"

Reply:
Done with the last commit.